docs/content/guides/author-apps/containers/overview/index.md

```diff
@@ -90,7 +90,7 @@ Refer to the probes section of the [container resource schema]({{< ref "containe
 
 ## Connections
 
-When a connection between two resources is declared, Radius injects resource related information into environment variables that are then used to access the respective resource without having to hard code URIs, connection strings, access keys, or anything that application code needs to successfully communicate.
+When a connection between two resources is declared, Radius injects resource related information into environment variables that are then used to access the respective resource without having to hard code URIs, connection strings, access keys, or anything that application code needs to successfully and securely communicate, which is done by storing these environment variables as secrets inside of the Radius Environment.
```

Comment:
Could you double check that we store environment variable values in a Kubernetes secret for both automatically-configured values, as well as manually specified values?

Reply:
I'd try this:

```bicep
resource container 'Applications.Core/containers@2023-10-01-preview' = {
  name: 'demo'
  properties: {
    application: application.id
    container: {
      image: 'ghcr.io/radius-project/samples/demo:latest'
      env: {
        foo: 'bar'
      }
    }
    connections: {
      redis: {
        source: redis.id
      }
    }
  }
}
```

and then check the Kubernetes deployment/pod definition to see where the values are stored.

Reply:
Hey tested out the automatically-configured values with @kachawla  and confirmed that those are stored as Kubernetes secrets, however manually specified values are not.

Example output:

```
kubectl get secret demo -n default-demo -o json
```

```
{
    "apiVersion": "v1",
    "data": {
        "CONNECTION_REDIS_CONNECTIONSTRING": "",
        "CONNECTION_REDIS_HOST": "cmVkaXMtcjV0Y3JyYTNkN3VoNi5kZWZhdWx0LWRlbW8uc3ZjLmNsdXN0ZXIubG9jYWw=",
        "CONNECTION_REDIS_PORT": "NjM3OQ=="
    },
    "kind": "Secret",
    "metadata": {
        "creationTimestamp": "2024-02-14T21:54:35Z",
        "labels": {
            "app.kubernetes.io/managed-by": "radius-rp",
            "app.kubernetes.io/name": "demo",
            "app.kubernetes.io/part-of": "demo",
            "radapp.io/application": "demo",
            "radapp.io/resource": "demo",
            "radapp.io/resource-type": "applications.core-containers"
        },
        "name": "demo",
        "namespace": "default-demo",
        "resourceVersion": "3095",
        "uid": "1d9740e0-3bee-44f1-bbe1-9dbe74559451"
    },
    "type": "Opaque"
}
```

Removed certain values

Reply:
@AaronCrawfis Should I just update the description to explicitly call out that its automatically-configured values only?

Reply:
We should do 2 things:

1. Keep this page implementation-agnostic, where we don't include any information about the runtime. Since Radius applications are platform-agnostic, we don't want to include information about Kubernetes when in the future we may have other runtimes that aren't Kubernetes. So on this page, we can say we that for Radius connections we securely store the environment variables.
2. On this page: https://docs.radapp.io/guides/operations/kubernetes/overview/#resource-mapping, we add information about how a connection creates a Kubernetes secret with a specific name and then uses that secret for the env var values.